atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/http/JepsenLockClientTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http;
+
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Set;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableSet;
+
+public class JepsenLockClientTest {
+    @SuppressWarnings("unchecked") // Mock in a test
+    private final JepsenLockClient<String> mockClient = (JepsenLockClient<String>) mock(JepsenLockClient.class);
+    private final StringLockClient client = new StringLockClient(mockClient);
+
+    private static final String LOCK_TOKEN = "foo";
+
+    @Test
+    public void unlockSingleDoesNotCallUnlockWithNull() throws InterruptedException {
+        client.unlockSingle(null);
+        verify(mockClient, never()).unlock(any());
+    }
+
+    @Test
+    public void unlockSingleCallsUnlockWithMatchingArgument() throws InterruptedException {
+        client.unlockSingle(LOCK_TOKEN);
+        verify(mockClient).unlock(eq(ImmutableSet.of(LOCK_TOKEN)));
+    }
+
+    @Test
+    public void unlockSingleReturnsTrueIfTokenCanBeUnlocked() throws InterruptedException {
+        when(mockClient.unlock(any())).thenReturn(ImmutableSet.of(LOCK_TOKEN));
```

Comment:
Right, yeah that makes sense for the idea that 'token can be unlocked / refreshed'. Changed.